keyboards/ergodox_ez/keymaps/dvorak_plover/rules.mk

```diff
@@ -0,0 +1,3 @@
+STENO_ENABLE = yes
+# Not enough interupts, so something has to go
+MOUSEKEY_ENABLE = no
```

Comment:
I found that you didn't need to disable mouse-keys to get this effect. See my Makefile here:

https://github.com/arxanas/qmk_firmware/blob/d315d539f9327407f403cf6d325b32fd97c09b03/keyboards/ergodox/ez/keymaps/plover-arxanas/Makefile#L3-L7

You might want to try it to see if it still works with the Gemini protocol.

```
# Not enough interrupts are available, so we have to disable something. I use
# mouse keys (MOUSEKEY_ENABLE) and the volume keys (EXTRAKEY_ENABLE), so I
# didn't want to disable any of those. This is some sort of debugging thing, I
# think, so I disabled it because I don't need it.
CONSOLE_ENABLE = no
```